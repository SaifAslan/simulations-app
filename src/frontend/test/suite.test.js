describe('suite name', () => {
  it('foo', () => {
    expect(Math.sqrt(4)).toBe(2);
  });

  it('bar', () => {
    expect(1 + 1).toBe(2);
  });

  it('snapshot', () => {
    expect({ foo: 'bar' }).toMatchSnapshot();
  });
});
