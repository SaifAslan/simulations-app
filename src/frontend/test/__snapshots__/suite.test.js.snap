// Jest Snapshot v1, https://jestjs.io/docs/snapshot-testing

exports[`suite name > snapshot 1`] = `
{
  "foo": "bar",
}
`;

exports[`suite name snapshot 1`] = `
{
  "foo": "bar",
}
`;
